src/frontend/shortlist/src/components/school/SchoolCard.vue

```diff
@@ -26,40 +27,44 @@ export default {
     <div class="school-simple-container" @click="swapDetail">
       <template v-if="!isDetail">
         <div class="school-simple-img-container">
-          <img
-            class="school-simple-img"
-            :src="schoolData.img"
-            draggable="false"
-          />
+          <img class="school-simple-img" :src="defaultImg" draggable="false" />
+          <!-- :src="schoolData.img" -->
         </div>
       </template>
 
       <div class="school-simple-name-row">
         <div class="school-simple-name-name">
           <template v-if="isDetail">
-            <a :href="schoolData.link">{{ schoolData.name }}</a>
+            <a :href="schoolData.school.schoolMetadata.link">{{
+              schoolData.school.schoolMetadata.name
```

Comment:
Could you please explain what this link is doing here? The link to the school website, or the picture that is still being worked on?